apps/omg/test/support/integration/deposit_helper.ex

```diff
@@ -39,7 +39,7 @@ defmodule Support.Integration.DepositHelper do
   end
 
   def deposit_to_child_chain(to, value, token_addr) when is_binary(token_addr) and byte_size(token_addr) == 20 do
-    contract_addr = Config.maybe_fetch_addr!(nil, :erc20_vault)
```

Comment:
`maybe_fetch_addr ` is one of the things that I'm most happy we could remove at this point.
snapshots provide as a consistent world view of contract addresses, no need for parametrization anymore